kuma/wiki/search.py

```diff
@@ -0,0 +1,255 @@
+# -*- coding: utf-8 -*-
+import operator
+
+from django.conf import settings
+from django.db.models import Q
+from django.utils.html import strip_tags
+
+from elasticsearch.helpers import bulk
+from elasticsearch_dsl import document, field
+from elasticsearch_dsl.connections import connections
+from elasticsearch_dsl.mapping import Mapping
+from elasticsearch_dsl.search import Search
+
+
+# Configure Elasticsearch connections for connection pooling.
```

Comment:
I think we should prevent accidentally calling this multiple times, how about putting that into a function and call it in the models.py? Then once we're on Django 1.7 we can move this to a AppConfig class's setup method